src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -312,14 +411,15 @@ def __call__(self, rcvd_messages: List[Tuple[int, Message]]):
         valid_proposal_orders[self.power_name] = orders
 
         best_proposer, best_orders = yield from get_best_orders(self, valid_proposal_orders, shared_orders)
-        best_orders, best_proposer = orders, list(self.alliances.keys())[0] if self.alliances else ""
 
         # add orders
         orders_data = OrdersData()
         orders_data.add_orders(best_orders)
-
         self.orders = orders_data
 
+        # filter out aggressive orders to allies
+        yield self.replace_aggressive_order_to_allies()
```

Comment:
I would highly prefer a pattern in which orders are passed into and out of this function, e.g.

`orders = yield self.replace_aggressive_order_to_allies(best_orders)`